### PR TITLE
Rasa X Annotations

### PR DESCRIPTION
**Pull request by b-quachtran**
Rasa X Annotations

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @melindaloubser1 will take a look at it as soon as possible ✨

**Comment by erohmensing** [comment]
goddamn it sorry about tagbot

**Comment by erohmensing** [comment]
Sorry about the merge conflicts 😅 had to merge it in eventually

File: data/nlu/nlu.md [hunk 1]
```diff
@@ -923,6 +923,9 @@
 - What is included in [RASA open source](product) edition?
 - you are [open source](product)
 - Is rasa opne source?
+- is rasa open
+- okay tell me about the pricing details
+- I want to buy rasa
```
Comment (melindaloubser1) [inline data/nlu/nlu.md:928]:
```suggestion
- I want to buy [rasa](product)
```

File: data/nlu/nlu.md [hunk 2]
```diff
@@ -923,6 +923,9 @@
 - What is included in [RASA open source](product) edition?
 - you are [open source](product)
 - Is rasa opne source?
+- is rasa open
```
Comment (melindaloubser1) [inline data/nlu/nlu.md:926]:
```suggestion
- is [rasa](product) open
```

**Pull request closed without merging**